docker/android-system.sh

```diff
@@ -53,6 +54,9 @@ EOF
     curl --retry 3 -sSfL https://storage.googleapis.com/git-repo-downloads/repo -O
     chmod +x repo
 
+    # the `repo` tool requires python3 so change the default python interpreter
```

Comment:
Can we call it with `python3 ./repo` instead?

Reply:
that works and it's better! updated the PR